### PR TITLE
Require round-to-nearest-even behavior for Ruby 2.4+

### PR DESCRIPTION
**Pull request by noahgibbs**
But require round-to-nearest before that.

### CODE REVIEW

File: core/float/round_spec.rb [hunk 1]
```diff
@@ -65,11 +65,23 @@
...
+      -2.5e20.round(-20).should   eql( -2 * 10 ** 20  )
+    end
+  end
+
+  ruby_version_is ""..."2.4" do
+    it "returns big values rounded to nearest" do
+      +2.5e20.round(-20).should   eql( +3 * 10 ** 20  )
+      -2.5e20.round(-20).should   eql( -3 * 10 ** 20  )
+    end
+  end
```
Comment (eregon) [inline core/float/round_spec.rb]: Could you swap these two ìt` so the 2.4 behavior comes after the < 2.4 behavior?

Comment (noahgibbs) [inline core/float/round_spec.rb]: Absolutely. Pushed.

**Comment by eregon** [comment]
It looks great, thanks for adapting the specs!
As soon as the change is in MRI we can merge this 😃

**Comment by Dorian** [comment]
Was reverted here: https://github.com/ruby/spec/commit/8570c4a081338485c60ec73870f38b5737ed2dc0

**Pull request merged**